integration/spark/src/test/scala/org/apache/carbondata/view/rewrite/TestAllOperationsOnMV.scala

```diff
@@ -610,10 +610,10 @@ class TestAllOperationsOnMV extends QueryTest with BeforeAndAfterEach {
     sql("drop materialized view if exists mv1")
     sql("create materialized view mv1  as select a.name,a.price from maintable a")
     var dataFrame = sql("select a.name,a.price from maintable a limit 1")
-    assert(dataFrame.count() == 1)
```

Comment:
why this change? because count is also an action, it should also give 1

Reply:
In this Case, count was returning 1 only, but df.show() is giving all results. In cluster also, limit query gives all rows. Because of this, testcase didnt catch this issue. That's why changed to df.collect.length

Reply:
Here, count API is not applying collect rows on SPARK PLAN. When i checked it, plan was on main table

Reply:
@akashrn5 Please help to review and merge